src/App/Transactions.cpp

```diff
@@ -281,16 +277,8 @@ TYPESYSTEM_SOURCE_ABSTRACT(App::TransactionObject, Base::Persistence)
 //**************************************************************************
 // Construction/Destruction
 
-/**
- * A constructor.
- * A more elaborate description of the constructor.
- */
 TransactionObject::TransactionObject() = default;
 
-/**
- * A destructor.
- * A more elaborate description of the destructor.
- */
 TransactionObject::~TransactionObject()

```

Comment:
Here we completely remove the destructor/constructor docstrings in opposition to above